test/raven.test.js

```diff
@@ -2130,6 +2130,23 @@ describe('install/uninstall', function () {
             Raven.install();
             assert.isTrue(TraceKit.report.subscribe.calledOnce);
         });
+
+        it('should use attachEvent instead of addEventListener in IE8', function () {
+            // Maintain a ref to the old function so we can restore it later.
+            var temp = document.addEventListener;
+
+            // Test setup.
+            this.sinon.stub(Raven, 'isSetup').returns(true);
+            document.addEventListener = false;
```

Comment:
I was unsure if there was a better way to do this. `this.sinon.stub(document, 'addEventListener', false)` didn't seem to do what I wanted.